NPM Install Fails Due to `rollup`

Output from running `npm install`:

```
created dist/index.cjs.js, dist/index.esm.js in 13.2s

dist/declaration/index.d.ts → dist/index.d.ts...
[!] RollupError: "mergeWith" is not exported by "node_modules/@types/lodash/index.d.ts", imported by "node_modules/@types/lodash.mergewith/index.d.ts".
https://rollupjs.org/troubleshooting/#error-name-is-not-exported-by-module
node_modules/@types/lodash.mergewith/index.d.ts
    at error (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:260:30)
    at Module.error (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:13652:16)
    at Module.traceVariable (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:14034:29)
    at ModuleScope.findVariable (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:12516:39)
    at Identifier.bind (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:8433:40)
    at ExportDefaultDeclaration.bind (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:6231:23)
    at Program.bind (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:6227:28)
    at Module.bindReferences (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:13648:18)
    at Graph.sortModules (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:24726:20)
    at Graph.build (/source/nautobot-ui/node_modules/rollup/dist/shared/rollup.js:24611:14)


ERROR: "rollup" exited with 1.
npm notice 
npm notice New minor version of npm available! 9.5.1 -> 9.6.5
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.6.5>
npm notice Run `npm install -g npm@9.6.5` to update!
npm notice 
npm ERR! code 1
npm ERR! path /source/nautobot-ui
npm ERR! command failed
npm ERR! command sh -c npm run build

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-04-25T18_48_29_827Z-debug-0.log
```

--- a/rollup.config.mjs
+++ b/rollup.config.mjs
@@ -7,7 +7,7 @@ import dts from 'rollup-plugin-dts';
 
 import pkg from './package.json' assert { type: 'json' };
 
-const external = Object.keys(pkg.peerDependencies);
+const external = [...Object.keys(pkg.peerDependencies), 'lodash.mergewith'];
 
 /** @type {import('rollup').RollupOptions[]} */
 export default [

